lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -54,14 +70,16 @@ def ask_for_repmgrd_configuration
 
     def activate
       say("Configuring Replication Standby Server...")
-      data_dir_empty? &&
+      initialize_postgresql_disk if disk
+      PostgresAdmin.prep_data_directory &&
+        data_dir_empty?
```

Comment:
No, this is wrong. If the directory is not empty, we want this to fail before doing any of this other work.

I was advocating removing the `&&` from after `PostgresAdmin.prep_data_directory` because it doesn't conform to the return value we would expect.

Then if `PostgresAdmin.prep_data_directory` fails we will just fail on `data_dir_empty?` and not execute the rest.